src/shared/utils/cache/get-enclave-wallet.ts

```diff
@@ -0,0 +1,61 @@
+import { thirdwebClient as client } from "../sdk";
+import { EnclaveWallet } from "thirdweb/dist/types/wallets/in-app/core/wallet/enclave-wallet";
+import { getUserStatus } from "thirdweb/dist/types/wallets/in-app/core/actions/get-enclave-user-status";
+import type { Ecosystem } from "thirdweb/dist/types/wallets/in-app/core/wallet/types";
+import { ClientScopedStorage } from "thirdweb/dist/types/wallets/in-app/core/authentication/client-scoped-storage";
```

Comment:
was there no way to just use `inAppWallet()` / `ecosytemWallet` for this?

can't really access these internal things like that. Might be worth exposing some of these if you need them, but probably with a bit higher level api

Reply:
Restating from internal for posterity: I have made the changes to use the preferred functions, however storage within the current design causes unintended results because it is not a browser environment, and will need to be addressed